content/docs/guides/crosswalk/kubernetes/aws/02-kubernetes-stacks/01-cluster-configuration.md

```diff
@@ -0,0 +1,672 @@
+---
+
+title: Kubernetes Cluster
+
+aliases: ["/docs/guides/crosswalk/kubernetes/aws/cluster/"]
+---
+
+Manage the control plane, worker nodes, RBAC, and defaults.
+
+  * [Create the Control Plane](#create-the-control-plane)
+  * [Create the Worker Nodes](#create-the-worker-nodes)
+  * [Use the Cluster](#use-the-cluster)
+  * [Setting Defaults](#setting-defaults)
+  * [Use Kuberntes RBAC](#use-kubernetes-rbac)
+  * [Updating the Worker Nodes](#updating-the-worker-nodes)
+
+
+The full code for this stack is on [GitHub][gh-repo-stack].
+
+See the [official Kubernetes docs][k8s-docs] for more details.
+
+## Create the Control Plane
+
+Create a new EKS cluster control plane equipped with:
+
+**Identity**
+
+  * IAM roles for each worker node group.
+  * IAM roles for users mapped into Kubernetes RBAC.
+
+**Networking**
+
+  * Public subnets for use with public load balancers.
+  * Private subnets for use with private load balancers.
+  * Private subnets used as the default subnets for workers, by design.
+  * Pod networking managed by the [AWS CNI Plugin][aws-k8s-cni].
+
+**Storage**
+
+  * Default Kubernetes storage classes backed by cloud volume types.
+
+**Workers**
+
+  * No default worker node group in favor of specific, separately managed node groups.
+
+**General**
+
+  * A specific version of Kubernetes for the control plane.
+  * Tags for resources under cluster management.
+  * Control plane logging enabled.
+  * Configurable accessibility for the cluster API endpoint (optional).
+
+    ```typescript
+    import * as eks from "@pulumi/eks";
+    
+    // Create an EKS cluster.
+    const cluster = new eks.Cluster(`${projectName}`, {
+        instanceRoles: [
+            aws.iam.Role.get("adminsIamRole", stdNodegroupIamRoleName),
+            aws.iam.Role.get("devsIamRole", perfNodegroupIamRoleName),
+        ],
+        roleMappings: [
+            {
+                roleArn: config.adminsIamRoleArn,
+                groups: ["system:masters"],
+                username: "pulumi:admins",
+            },
+            {
+                roleArn: config.devsIamRoleArn,
+                groups: ["pulumi:devs"],
+                username: "pulumi:alice",
+            },
+        ],
+        vpcId: config.vpcId,
+        publicSubnetIds: config.publicSubnetIds,
+        privateSubnetIds: config.privateSubnetIds,
+        storageClasses: {
+            "gp2": { type: "gp2", encrypted: true},
+            "sc1": { type: "sc1"}
+        },
+        nodeAssociatePublicIpAddress: false,
+        skipDefaultNodeGroup: true,
+        deployDashboard: false,
+        version: "1.14",
+        tags: {
+            "Project": "k8s-aws-cluster",
+            "Org": "pulumi",
```

Comment:
How I wish we had a `getOrganizationName()`! (https://github.com/pulumi/pulumi/issues/2800)